src/cryptoadvance/specter/wallet.py

```diff
@@ -402,6 +402,37 @@ def fetch_transactions(self):
                 self._addresses.add(addresses, check_rpc=False)
                 self._addresses.add(change_addresses, check_rpc=False)
 
+    def import_electrum_label_export(self, electrum_label_export):
+        if not electrum_label_export:
+            logger.warning(f"No electrum export json provided.")
+            return
+
+        labeled_addresses = json.loads(electrum_label_export)
+
+        # write tx_label to address_label in labels
+        for txitem in self._transactions.values():
+            if txitem["txid"] not in labeled_addresses:
+                continue
+
+            address_list = (
+                [txitem["address"]]
+                if isinstance(txitem["address"], str)
+                else txitem["address"]
+            )
```

Comment:
By the way: I noticed that `txitem["address"]` is a `str` if there is only 1 output address that belongs to the wallet. However if there are multiple addresses that belong to the wallet the type is `list`.  This seems not ideal.  Is there any plan to change that to a `list` always in the future?